lirp-sql/src/integrationTest/kotlin/net/transgressoft/lirp/persistence/sql/UuidJdbcInteropIntegrationTest.kt

```
@@ -0,0 +1,191 @@
+                            affected shouldBe 0
+                        }
+                    }
+                    "H2", "PostgreSQL" -> {
+                        val affected =
+                            try {
+                                conn.prepareStatement("UPDATE uuid_interop_t SET payload = ? WHERE id = ?").use { ps ->
+                                    ps.setString(1, "via-string")
+                                    ps.setString(2, uuid.toString())
+                                    ps.executeUpdate()
+                                }
+                            } catch (e: SQLException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)